src/App.jsx

```diff
@@ -1,5 +1,48 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import classNames from 'classnames';
+
+const clearButton = (setCurrentSelected) => (
+  <button 
+      data-cy="ClearButton" 
+      type="button" 
+      
+      onClick={() => 
+      {
+        setCurrentSelected('')
+      }}
+      className ="delete ml-3"
+      />
+)
+
+const RemoveButton = ({setCurrentSelected}) => (
+  <button 
+    data-cy="RemoveButton" 
+    type="button" 
+    className= {classNames("button", 'is-info')}
+    onClick = {() => setCurrentSelected('')
+      }
+      >
+      -
+    </button>
+)
+
+const AddButton = ( {good,setCurrentSelected}) => (
+  <button 
+              data-cy="AddButton" 
+              type="button" 
+              className= {classNames("button", )}
```

Comment:
The `className` property in the `AddButton` component is missing a class name. Ensure that you provide a valid class name for styling purposes.